api-specificatie/openapi.yaml

```diff
@@ -202,7 +202,7 @@ paths:
     get:
       security:
         - apiKeyBAG: []
-      operationId: raadpleegAdresseerbaarobjectMetAdoId
+      operationId: raadpleegAdresseerbaarobject
```

Comment:
je hebt 'raadpleegAdres' gewijzigd naar 'raadpleegAdresMetNumId' en 'raadpleegAdresseerbaarobjectMetAdoId' gewijzigd naar 'raadpleegAdresseerbaarobject'. De inconsistentie is hiermee omgedraaid maar denk ik niet opgelost.
Dit is denk ik consequentie van 'Bij het bevragen met een ID van een resource anders dan degene die wordt opgevraagd, operation ID raadpleegId'. Maar als ik dat niet had gelezen, had ik vooral gedacht dat het erg inconsequent is. Dus als het je bedoeling was om het minder inconsequent te maken, vraag ik me af of dat hiermee gelukt is.

Het is voor gebruikers denk ik niet zo belangrijk hoe een operationId heet, het moet enigszins duidelijk zijn en vooral uniek (geen twee operationId's in zelfde API mogen gelijk zijn), dus vind ik het verder prima om het zo te maken als jij wilt.

Reply:
Dan laat ik het graag zo.